package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Not sure of best way to do this... these packages are from [here](https://github.com/praekelt/mama-ng-deploy/blob/develop/docker-vms/jssandbox/Dockerfile#L8).

The versions (where possible) are from [here](https://github.com/praekelt/vumi-go/blob/develop/package.json#L11-L23).